py/torch_tensorrt/dynamo/partitioning/common.py

```diff
@@ -35,8 +35,12 @@ def construct_dynamic_input(
             node = dim.node
             expr = node.expr
             shape_env = node.shape_env
-            var_range = shape_env.var_to_range.get(expr, None)
-            var_val = shape_env.var_to_val.get(expr, None)
+            var_range = shape_env.var_to_range.get(expr, None) or shape_env.bound_sympy(
+                expr
+            )
+            var_val = shape_env.var_to_val.get(expr, None) or expr.xreplace(
+                shape_env.var_to_val
+            )
```

Comment:
Does this line prompt evaluation of the symbolic expression, like `s0*s1`, which has been mentioned in the PR summary? Could a comment be added to the above two lines to describe their effect on the symbolic expressions

Reply:
added it now.